Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp

```diff
@@ -3073,12 +3084,20 @@ void CanvasRenderingContext2DBase::setWordSpacing(const String& wordSpacing)
     CSSTokenizer tokenizer(wordSpacing);
     auto tokenRange = tokenizer.tokenRange();
     tokenRange.consumeWhitespace();
-    RefPtr parsedValue = CSSPropertyParserHelpers::consumeLength(tokenRange, HTMLStandardMode);
+
+    auto parsedValue = CSSPropertyParserHelpers::MetaConsumer<CSS::Length<>>::consume(tokenRange, HTMLStandardMode, { }, { .unitlessZero = UnitlessZeroQuirk::Allow });
     if (!parsedValue)
         return;
-    modifiableState().wordSpacing = parsedValue->cssText();
+    auto rawLength = parsedValue->raw();
+    if (!rawLength)
+        return;
+
+    // FIXME: Ensure rawLength->unit is supported by `Style::computeUnzoomedNonCalcLengthDouble`.
```

Comment:
Dows this shortcoming have test coverage?

Reply:
It’s an existing problem I am fixing in a follow up PR with tests.